lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -559,6 +561,23 @@ SPIRVConstant *SPIRVModuleImpl::getLiteralAsConstant(unsigned Literal) {
   return V;
 }
 
+SPIRVInstruction *
+SPIRVModuleImpl::getVerifiedInsertBefore(SPIRVInstruction *RequestedInsertBefore) {
+  if (RequestedInsertBefore)
```

Comment:
I'd suggest to use `{` here to improve readabilty.

Reply:
Good point, done